packages/client/src/i18n/messages/user.ts

```diff
@@ -458,6 +462,33 @@ const messagesToDefine: IUserMessages = {
     id: 'system.user.settings.avatarUpdated',
     defaultMessage: 'Profile image successfully updated',
     description: 'Profile image change message on success'
+  },
+  name: {
+    id: 'system.user.settings.name',
+    defaultMessage: 'Name',
+    description: 'label for Name'
+  },
+  systemLanguage: {
+    id: 'system.user.settings.systemLanguage',
+    defaultMessage: 'System language',
+    description: 'Label for system language'
+  },
+  profileImage: {
+    id: 'system.user.settings.profileImage',
+    defaultMessage: 'Profile Image',
+    description: 'Label for profile image'
+  },
+  imageFormat: {
+    id: 'imageUploadOption.upload.imageFormat',
+    defaultMessage:
+      'Image format not supported. Please attach a png or jpg (max 5mb)',
+    description:
+      'Show error message if the selected image type is not supported'
+  },
+  overSized: {
+    id: 'imageUploadOption.upload.overSized',
+    defaultMessage: 'File is too large. Please attach file less than 5mb',
+    description: 'Error message for Attachment size greater than 5mb.'
```

Comment:
These might have been duplicated

Reply:
Yeah, they are already in imageUploader messages.